default.properties

```diff
@@ -55,6 +55,17 @@ pgsqlUser=georchestra
 # Default common password of PostgreSQL user for all geOrchestra modules
 pgsqlPassword=georchestra
 
+### rabbitmq properties
+
+# rabbitmq server domain name
+rabbitmqHost=${RABBITMQHOST}
```

Comment:
Like discussed in slack, you need to change to:
```
rabbitmqHost=${RABBITMQ_HOST}
rabbitmqUser=${RABBITMQ_USERNAME}
rabbitmqPassword=${RABBITMQ_PASSWORD}
```

Thank you

Reply:
To be taken into account, thanks.